tests/test_github_interactions.py

```diff
@@ -124,8 +128,8 @@ def test_post_all(test_data):
     """Check that all collected entries from config can be posted."""
     # create github manager object with the directory as argument
     manager = github_interaction.GithubManager()
-    manager.api = mock_gh_api.MockGH()
     manager.parse_config_list(test_data["collect_config_test"]["postable_sample"])
+    manager.api = mock_gh_api.MockGH()
```

Comment:
I'm curious about the reasoning behind changing the order of this statement. Can you please explain that.